app/classes/api/sequence_api.rb

```diff
@@ -11,7 +11,6 @@ class SequenceAPI < ModelAPI
     self.delete_page_length      = 1000
 
     self.high_detail_includes = [
-      :observation,
```

Comment:
Again it's not clear to me if this is removing information from an API call that clients might be expecting.

Reply:
Yes, I am removing details of the observation from this query.  I couldn't imagine any plausible use-case where it would ever happen.  The typical case will be to go the other direction: Ask for observations that match your criteria and that have sequences.  And in fact, high-detail responses for the observations end point include the basic sequence info (locus, bases, etc.)  An alternative way to go would be to ask just for the ids of observations matching your criteria (detail = none), and then ask for sequences for those observations.  This returns only the information you want if you are only interested in sequences.  In neither case does it make any sense to return a bunch of details about the observation when querying sequences.

I think there are a few other cases like this, too.  Collection numbers and herbarium records and external links come to mind.  These are all very specialized things that users will probably rarely query directly, or will ask for once they already have the observations.  By doing this I'm subtly encouraging people to use observations as the entry point for just about everything.  Note that, for example, the interface for querying sequences and collection numbers and comments and even images does not allow you to filter by species name or location or anything helpful.  If you want to do that you must go through observations.  I know Query allows some of that, but those queries in general involve joins and some become quite expensive (especially images joined to anything via observations and images_observations).  For the website UI it made sense to allow that to a limited degree.  But for the API, by definition, users will be programmers, and as such will be perfectly capable of doing these joins themselves.  So they are encouraged to do their fancy queries on observations first, and either get the info about the attached objects like sequences and comments and images via high detail response of the original observations query, or by asking for the matching sequences, comments, images, etc. in subsequent queries to each of those endpoints.

You could argue "what's the harm in keeping the observation details in there", but observations are *huge*, there are so many fields associated with observations.  So in the spirit of pairing things down to just what's actually used and speeding everything up, out went the observation details.